includes/_top.htm

```diff
@@ -1,5 +1,55 @@
 <body>
 	<div class="main">
-		<div class="header">
+		<!-- <div class="header">
 			<a href="/"><img width="325" height="97" class="logo" src="/images/logo.png"></a>
+			        
+		</div> -->
+				<div class="head">
+				<table>
+				  <tr>
+				    <th><a href="/"><img width="325" height="97" class="logo" src="/images/logo.png"></a></th>
+				    <th></th>
```

Comment:
what's with all the empty headers? is this just for spacing or something? can we use css instead?

Reply:
it's definitely for spacing, and could definitely be handled with CSS more elegantly - however,  that's outside of my level of knowledge. i was just trying to hack something together quickly. CC: @rajinisivaram

Reply:
![image](https://user-images.githubusercontent.com/11722533/30495284-4c70bd70-9a00-11e7-88a5-01db7b262269.png)